src/com/owncloud/android/lib/resources/files/CreateRemoteFolderOperation.java

```diff
@@ -99,6 +99,7 @@ private RemoteOperationResult createFolder(OwnCloudClient client) {
         MkColMethod mkcol = null;
         try {
             mkcol = new MkColMethod(client.getWebdavUri() + WebdavUtils.encodePath(mRemotePath));
+            client.setUseNextcloudUserAgent(true);
```

Comment:
Hm? But this will not work on pre 12.0.5 and definitely not on 11? Or am I wrong?

Reply:
I tried it with NC11:
- login with a fresh installation
- create folder
- up&download file

All is working. I suspect that not on every call the user agent is checked.
Shall we ask some server guy on which calls exactly this is done?

Reply:
Yes please.

Reply:
According to @Morris this is used widely across NC:
notifications, ransomware_protection, globalsiteselector (login), files_downloadactivity, external (server side menu items), workflow engine and DAV auth

Back to 11.0.7 all versions have ownCloud/Nextcloud user agent support. 
So only those really outdated NC10 or not updated NC11 version will have a problem.

Reply:
Unfortunately there are still people with Nc10 (3.x not working reported Nc10) but I'd be fine with dropping 10 since it is EOL. So that is enough for the argument that we don't support EOL server versions.

Reply:
Can we merge this? @mario @AndyScherzinger 
I want to do 3.0.2 today.

Reply:
:+1: sure

Reply:
👍